src/modules/universe/RadixNodeConnection.ts

```diff
@@ -322,7 +322,7 @@ export class RadixNodeConnection extends events.EventEmitter {
     }
 
     private _onAtomReceivedNotification = (notification: AtomReceivedNotification) => {
-        logger.debug('Atoms notification', notification)
+        // logger.debug('Atoms notification', notification)
```

Comment:
Old sins, I know, but maybe consider removing dead code? (thinking of the below 10 or so lines)

Reply:
So in this particular case I'm going to keep it as is, because it's not dead code, it's just disabled code. Half the bugs I encounter require me to come down here and turn it on to help figure out what's happening. Having to rewrite the atom storing from scratch every time doesn't seem like a good deal.